kerastuner/engine/host.py

```diff
@@ -14,15 +14,14 @@
 
 from __future__ import absolute_import
 
-from .state import State
 from ..abstractions.display import fatal, subsection, warning
 from ..abstractions.display import display_settings
 from ..abstractions.tensorflow import TENSORFLOW as tf
 from ..abstractions.tensorflow import TENSORFLOW_UTILS as tf_utils
 from .. import config
 
 
-class Host(object):
+class Host:
```

Comment:
All objects should inherit from object, no?

Reply:
Only if you want to be compatible with Python 2. In python 3 this is optional.

Reply:
We should seek to avoid any kind of straightforwardly fixable Py2 incompatibility, since Py2 is still widely used at Google. We don't have Python2 support as an explicit goal today, but avoiding incompatibilities like this will reduce work when we decide to support it.

Reply:
Tensorflow will drop Python 2 in 5 months: https://github.com/python3statement/python3statement.github.io/pull/179#issue-251778069
As well as numpy, scipy... etc.

and in the readme it's said that python 3.6+ is a requirement. It's quite misleading.

Shoud I add another entry in travis to test again python 2.7? We should avoid confusion.

Reply:
As, Python 2 support is not a goal at this time and so we don't need to test it. But let's keep `object` here.